packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -529,19 +543,26 @@ export class ScmResourceComponent extends ScmElement<ScmResourceComponent.Props>
 
     override render(): JSX.Element | undefined {
         const { hover } = this.state;
-        const { model, treeNode, colors, parentPath, sourceUri, decoration, labelProvider, commands, menus, contextKeys, caption } = this.props;
+        const { model, treeNode, colors, parentPath, sourceUri, decoration, labelProvider, commands, menus, contextKeys, caption, isLightTheme } = this.props;
         const resourceUri = new URI(sourceUri);
 
+        const [light, dark] = [treeNode.decorations?.icon, treeNode.decorations?.iconDark];
+        const themedIcon = isLightTheme ? light : dark;
+        const codiconClass = typeof themedIcon === 'string' && !themedIcon.includes('/') ? themedIcon : '';
```

Comment:
This check seems a little hakish:

```ts
!themedIcon.includes('/') 
```

Reply:
It is.
Now that `light` and `dark` are strings. I can check for `"hostedPlugin"` in the string. It will certainly be specific enough to avoid something else being caught. Hopefully general enough to understand it is supposed to be a path to a directory.